pyvera/__init__.py

```diff
@@ -293,13 +307,7 @@ def set_value(self, name, value):
                         'action': 'Set' + name,
                         'newLoadlevelTarget': value}
                 else:
-                    payload = {
```

Comment:
The `new{name}Value` isn't much of a standard from what I can see in the docs - although it does work for `Target` and `Armed`, both of which we use.

`new{name}` is perhaps used more after your additions.

I think I'd be tempted to have a list of `exceptions` and handle both in the base class. 

If there are other examples like the `curtain` where the same variable is re-used, this will handle it without further work.

But don't have strong feelings (except that Vera should have stuck to one standard!).